src/examples/StreetCanyon/Input/StreetCanyon.xml

```diff
@@ -0,0 +1,1486 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This scenario is pretty fast. If you want, you can set referencePoint to remove the backside. I did not check this but you can definitely use this to avoid the triangles.

Reply:
since the clipping is a ball you would risk to lose some valuable triangles. I would manually remove them instead. Is this scenario taken from a real street canyon (using, e.g., OSM) or was this done by hand?
If it's a real street canyon, ok, otherwise i would reduce it to the bare minimum

Reply:
This is taken by OSM. If you want, you can modify this scenario.